crates/uv-build-backend/src/source_dist.rs

```diff
@@ -122,13 +120,19 @@ fn source_dist_matcher(
 
     // Include the data files
     for (name, directory) in settings.data.iter() {
+        let directory = uv_fs::normalize_path(Path::new(directory));
```

Comment:
I think this was my main concern here: whether path normalization could muck with the glob at all. Taking a look at `normalize_path` itself, I don't _think_ so. At least, I can't see a way for it to go bad.